assets/blocks/course-list-block/index.js

```diff
@@ -3,10 +3,14 @@
  */
 import { __ } from '@wordpress/i18n';
 import { registerBlockVariation } from '@wordpress/blocks';
-import { list } from '@wordpress/icons';
 import { select, subscribe } from '@wordpress/data';
 import { addFilter } from '@wordpress/hooks';
 
+/**
+ * Internal dependencies
+ */
+import icon from '../..//icons/course-list-icon.svg';
```

Comment:
There's an extra / here. 🙂

Reply:
Thank you for the review Donna! I've fixed the issues here https://github.com/Automattic/sensei/pull/5595/commits/d47acada741b9121c02d0c68559fd8ae8b74311a :) 

I've also checked the PHP test failing issue. Looks like it is happening only for the nightly, and based on where it's failing, I'm not sure it's related to the codes on this PR. But I'll look a bit more into it separately.
![image](https://user-images.githubusercontent.com/6820724/188940730-a8ea80dc-2587-4480-a34c-9d6399c8b7e0.png)

Reply:
@gabrielcaires is also looking into this as it's happening everywhere. There appears to be something broken in our feature branch.